test/format/config.c

```diff
@@ -285,6 +285,23 @@ config_table(TABLE *table, void *arg)
             config_single(table, "btree.value_min=20", false);
     }
 
+    /*
+     * Limit the rows to 1000000 if the realloc exact and realloc malloc configs are on and not all
```

Comment:
We use the macros so that seeing the zeros doesn't require mental energy to count them. Similarly for here I'd suggest:
"Limit the rows to one million if the..."

Reply:
done